### PR TITLE
Add fingerprints of TOTEM Door Lock for SmartThingsEdgeDriver

### PR DESCRIPTION
**Pull request by FrankSpringfield**
The fingerprints of TOTEM Door Lock have already existed in the DTH(SmartThings Public). 
I tried to manage to migrate the fingerprints from DTH to Edge Driver.

Zigbee Device ID: "TOTEM H60/H90" & "TOTEM P30"

### CODE REVIEW

**Comment by lelandblue** [comment]
Hello, at this time we are only accepting new fingerprint submissions as part of the WWST Works With SmartThings Program. If you work for a device company and would like your device WWST Certified please reach out to partners@smartthings.com or check out more information [ here. ](https://developer.smartthings.com/docs/certification/overview)

**Comment by FrankSpringfield** [comment]
Hello, I'm now an intern working at SRC-N. The TOTEM fingerprints already exist in the DTH(SmartThings Public).  My leader wants me to migrate the fingerprints from DTH to Edge Driver.

**Comment by lelandblue** [comment]
Hey @FrankSpringfield Can you provide the JIRA ticket that your working from so we can have more discussion internally?

**Comment by FrankSpringfield** [comment]
Hi, the JIRA ticket is here: https://smartthings.atlassian.net/browse/WWST-8009.

**Review by wkhenon — APPROVED** [review]
This looks good to me

File: drivers/SmartThings/zigbee-lock/fingerprints.yml [hunk 1]
```diff
@@ -124,6 +124,17 @@ zigbeeManufacturer:
     manufacturer:
     model: E261-KR0B0Z0-HA
     deviceProfileName: lock-battery
+  # TOTEM
+  - id: TOTEM/H60/H90
+    deviceLabel: TOTEM Door Lock
+    manufacturer: TOTEM
+    model: H60/H90
+    deviceProfileName: lock-battery
```
Comment (lelandblue) [inline drivers/SmartThings/zigbee-lock/fingerprints.yml:132]: Does this device support the lockCodes capability ? If so  - the profile should be updated.

**Review by lelandblue — CHANGES_REQUESTED** [review]

File: drivers/SmartThings/zigbee-lock/fingerprints.yml (hunk 1 above)
Comment (lelandblue) [reply drivers/SmartThings/zigbee-lock/fingerprints.yml:132]: @FrankSpringfield This comment will need a response or on a related ticket before we can proceed. Thank you.

Comment (FrankSpringfield) [reply drivers/SmartThings/zigbee-lock/fingerprints.yml:132]: @lelandblue 
I'm sorry for responding too late. I have a day off yesterday.

As for the answer:
Yep, this device does support the lockCodes capability.
I'll change the deviceProfileName from lock-battery to base-lock and test whether it works well.

**Comment by FrankSpringfield** [comment]
Hi, @lelandblue.
Considering that the 'base-lock' profile has one additional capability, 'lockCodes', compared to the 'lock-battery' profile, in this commit I have changed the deviceProfileName from 'lock-battery' to 'base-lock'. 
After testing, we confirmed that the TOTEM H60 can be onboarded correctly through SmartThings after updating the edge driver.

**Review by greens — APPROVED** [review]

**Review by lelandblue — APPROVED** [review]

**Pull request merged**